core/templates/components/forms/custom-forms-directives/html-select.component.ts

```diff
@@ -27,16 +27,26 @@ export class HtmlSelectComponent implements OnInit {
   // and we need to do non-null assertion. For more information, see
   // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
   @Input() options!: { id: string; val: string }[];
-  @Input() selection!: string;
+  @Input() selectionID!: string;
```

Comment:
Normally we use selectionId (lowerCamelCase)

Reply:
Thanks, done.